.github/workflows/bump-opbeans-node.yml

```diff
@@ -0,0 +1,37 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it and tag this repo with that version, e.g.
+## "v1.2.3".
+name: Bump opbeans node
+
+on:
+  schedule:
+    - cron:  '0 20 * * 6'
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@v2
+      - name: Run Updatecli
+        run: updatecli diff --config ./.updatecli/bump-opbeans-node.yml
+      - name: Check git-diff
+        id: git-diff
+        run: git diff --exit-code --name-only --diff-filter=M --ignore-all-space || echo "diff=true" >> $GITHUB_OUTPUT
+      - name: Get new version
+        if: steps.git-diff.outputs.diff
+        run: echo "NEW_AGENT_VERSION=$(jq -r '.dependencies."elastic-apm-node".version')" >> $GITHUB_ENV
+      - name: Add, commit and push
+        if: steps.git-diff.outputs.diff
+        uses: EndBug/add-and-commit@v9.1.1
+        with:
+          add: package.json package-lock.json Dockerfile
+          message: |-
+            fix(package): bump elastic-apm-node to v${{ env.NEW_AGENT_VERSION }}
+          tag: v${{ env.NEW_AGENT_VERSION }}
```

Comment:
this can be done in the updatecli pipeline

Reply:
https://github.com/elastic/opbeans-node/pull/179/commits/eb1b46e35b4e9d36878077ef5e166598dc32f27a